server/core/src/main/java/dev/slimevr/bridge/ProtobufBridge.java

```diff
@@ -198,7 +198,7 @@ protected void trackerAddedReceived(TrackerAdded trackerAdded) {
 			remoteTrackersBySerial.put(tracker.getName(), tracker);
 		}
 		synchronized (remoteTrackersByTrackerId) {
-			remoteTrackersByTrackerId.put(tracker.getId(), tracker);
```

Comment:
I think this is generally supposed to be persistent, but I don't think we are entirely able to do that right now. Why not use ID though?